judge/contest_format/ioi16.py

```diff
@@ -0,0 +1,95 @@
+from django.db import connection
+from django.utils.translation import gettext_lazy
+
+from judge.contest_format.ioi import IOIContestFormat
+from judge.contest_format.registry import register_contest_format
+from judge.timezone import from_database_time
+
+
+@register_contest_format('ioi16')
+class IOI16ContestFormat(IOIContestFormat):
+    name = gettext_lazy('IOI16')
+    config_defaults = {'cumtime': False}
+    '''
+        cumtime: Specify True if time penalties are to be computed. Defaults to False.
```

Comment:
I know this is copied over from other contest formats, but I've never really liked the name. `use_time_penalty` or `time_penalty` seems more explicit.

Reply:
Given that this class inherits from IOIContestFormat, this would imply some other fixes (like overriding the other methods as they use `cumtime` as well). Is that what we want here ?
I think that to be consistent, either all should be changed to `use_time_penalty`, or none.

Reply:
Ah, you're right, I missed the inheritance. Let's not do that then. I wouldn't be opposed to changing this name everywhere in a later PR, though :+1: